src/pybind/mgr/dashboard/controllers/crush_rule.py

```diff
@@ -54,7 +54,11 @@ class CrushRuleUi(CrushRule):
     @ReadPermission
     def info(self):
         '''Used for crush rule creation modal'''
+        osd_map = mgr.get_osdmap()
+        crush = osd_map.get_crush()
+        crush.dump()
         return {
             'names': [r['rule_name'] for r in mgr.get('osd_map_crush')['rules']],
-            'nodes': mgr.get('osd_map_tree')['nodes']
+            'nodes': mgr.get('osd_map_tree')['nodes'],
+            'roots': crush.find_roots()
         }
```

Comment:
If we want to keep this `crush_rule` endpoint as a legacy one ok, but I'd suggest creating a new endpoint called `crush_tree` and expose the information as it's going to be consumed by the front-end: that is, in tree-like form (what we display in the Dashboard is not the CRUSH map or CRUSH rules, but the CRUSH tree).